_docs/c-advanced/tags.html

```diff
@@ -98,12 +98,12 @@
         </tr>
         <tr>
           <th>#DOW#</th>
-          <td>Day of week int (0-6)</td>
+          <td>Day of week int (0-6) (0-Monday)</td>
           <td>2</td>
         </tr>
         <tr>
           <th>#DOWB#</th>
-          <td>Day of week int (1-7)</td>
+          <td>Day of week int (1-7) (1-Monday)</td>
```

Comment:
@Melllin Thanks for these updates.  They definitely show that there's a lot of information we could add to make these docs more helpful to designers.

Just one suggested change before merging in.  How about a format like `(1-7 where 1 is Monday)` for the 2 above edits?